sources/iTermSemanticHistoryController.m

```diff
@@ -287,9 +290,11 @@ - (BOOL)openPath:(NSString *)path
         return YES;
     }
 
-    if (![self.fileManager fileExistsAtPath:path isDirectory:&isDirectory]) {
-        DLog(@"No file exists at %@, not running semantic history", path);
-        return NO;
+    if (isRawCoProcess) {
+        DLog(@"Launch coproress with script %@", script);
+        assert(delegate_);
+        [delegate_ semanticHistoryLaunchCoprocessWithCommand:script];
+        return YES;
     }
 
     if ([prefs_[kSemanticHistoryActionKey] isEqualToString:kSemanticHistoryCommandAction]) {
```

Comment:
Moving the test for `fileExistsAtPath:isDirectory:` below this changes the **Command** to run even when the file doesn't exist, which seems like an unnecessary change.